### DISCUSSION TITLE
help how to install java 17 for 1.16.5

### DISCUSSION THREAD

**Discussion by segvyfault** [FabricMC/fabric-api#2096] [Player support]
I installed java 17 from discord, searched everywhere on youtube but it doesn't work hel
![unknown](https://user-images.githubusercontent.com/92729699/159973242-112c2b37-4ecb-4cd4-9f38-ee541f309459.png)

**Comment by warjort** [comment]
Unobtainables 0.1.0 is for Minecraft 1.18.1 not 1.16.5
https://www.curseforge.com/minecraft/mc-mods/unobtainables/files

To answer your question more directly, changing the java version is a function of whatever minecraft installer you use.
